src/ChatUI/MainWindow.xaml.cs

```diff
@@ -31,6 +31,13 @@
 using MASES.DataDistributionManager.Bindings;
 using MASES.DataDistributionManager.Bindings.Configuration;
 using MASES.S4I.ChatLib;
+using System.IO;
+using System.Windows.Controls;
+#if NET
+using System.Windows.Forms;
+#else
+using Microsoft.Win32;
+#endif
```

Comment:
Why this?

Reply:
This was a legacy of a test, I removed it.